ros_ign_gazebo/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros_ign_gazebo</name>
+  <version>0.0.0</version>
+  <description>Tools for using Ignition Gazebo simulation with ROS.</description>
+  <maintainer email="ahcorde@openrobotics.org">Alejandro Hernandez</maintainer>
+
+  <license>Apache 2.0</license>
+
+  <author>Alejandro Hernandez</author>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>rclcpp</depend>
+  <depend>ignition-math6</depend>
+  <depend>ignition-msgs</depend>
```

Comment:
We won't have generic rosdep keys for Ignition libraries, so we always need to specify the version:

```suggestion
  <depend>ignition-msgs5</depend>
```